docs/secondary-network.md

```diff
@@ -0,0 +1,142 @@
+OVS bridge on based on the configuration. For example, the following
+configuration will create an OVS bridge named `br-secondary`, with a physical
+interface `eth1`.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: antrea-config
+  namespace: kube-system
+data:
+  antrea-agent.conf: |
+    secondaryNetwork:
+    ovsBridges: [{"bridgeName": "br-secondary", "physicalInterfaces": ["eth1"]}]
+```
+
+At the moment, Antrea supports only a single OVS bridge for secondary networks,
+and supports only a single physical interface on the bridge. The physical
+interface cannot be the Node's management interface, otherwise a Node's
+management network connectivity can be broken after `antrea-agent` creates the
+OVS bridge and moves the management interface to the bridge.
+
+## Secondary VLAN network configuration
+
+A secondary VLAN network is defined by a NetworkAttachmentDefinition CR. For
+example, the following NetworkAttachmentDefinition defines a VLAN network
+`vlan100`.
+
+```yaml
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: vlan100
+spec:
+  config: '{
+      "cniVersion": "0.3.0",
+      "type": "antrea",
+      "networkType": "vlan",
+      "mtu": 1500,
+      "vlan": 100,
+      "ipam": {
+        "type": "antrea",
+        "ippools": ["vlan100-ipv4", "vlan100-ipv6"]
+      }
+    }'
+```
+
+`antrea-agent` will connect a Pod secondary interface of a VLAN network to the
+secondary OVS bridge on the Node. If a non-zero VLAN is speficied in the
+network's `config`, `antrea-agent` will configure the VLAN ID to the OVS port,
+so the interface's traffic will be isolated within the VLAN. And before the
+traffic is forwarded out the Node via the secondary bridge's physical interface,
+OVS will insert the VLAN tag to the packets.
+
+A few extra notes about the NetworkAttachmentDefinition `config` fields:
+
+* `type` - must be set to `antrea`.
+* `networkType` - the only supported network type is `VLAN` as of now.
+* `mtu` - defaults to 1500 is not set.
+* `vlan` - can be set to 0 or a valid VLAN ID (1 - 4094). Defaults to 0. Also
+note, VLAN can be speficied in the spec of an IPPool in the `ipam` section too;
+but `vlan` in NetworkAttachmentDefinition `config` will override the VLAN in
+IPPool(s) if both are set.
+* `ipam` - it is optional. If not set, the secondary interfaces created for the
+network won't have an IP address allocated. For more information about secondary
+network IPAM configuration, please refer to the [Antrea IPAM document](antrea-ipam.md#ipam-for-secondary-network).
+
+## Pod secondary interface configuration
+
+You can create a Pod with secondary network interfaces by adding the
+`k8s.v1.cni.cncf.io/networks` annotation to the Pod. The following example Pod
+includes two secondary interfaces, one in network `vlan100` which should be
+created in the same Namespace as the Pod, the other in network `vlan200` which
+is created in Namespace `default`.
```

Comment:
nit: given that the Namespace of the Pod is actually `default`, I would recommend using a different Namespace (either for the Pod or for the network), to emphasize that cross-Namespace references are possible.

Reply:
Changed to `networks`. Let me know if you have a better Namespace name.